### DISCUSSION TITLE
Which port is used when connecting Temporary Wi-Fi hotspot?

### DISCUSSION THREAD

**Discussion by lp85d** [Mygod/VPNHotspot#571] [Q&A]
I launched the hotspot, and as I understand it, I need to register a proxy on Windows, but I only see the ip. and what port does this network have?
Redmi Note 8 Pro Russia MIUI V12.5.3.0.RGGRUXM 
I am using version 2.17.8 VPNHotspot

**Comment by lp85d** [comment]
And here's another question: when I connect to the hotspot, how can I control my smartphone via the ADB console?

**Reply by Mygod** [reply]
Not supported.

**Comment by Mygod** [comment]
This app does not require manual configuration of proxies on clients.
